tests/integration/test_integratord/data/test_cases/cases_integratord_change_inode_alert.yaml

```diff
@@ -1,4 +1,4 @@
-- name: Cannot read alerts - Inode changed
+- name: alerts_file_inode_changed
```

Comment:
Is there a reason to change each name? Is it a new change in the sintaxis?

Reply:
@damarisg this is because that name will be used after by pytest to assign an ID to the current test case. So, if we use the name given before my change, we won't be able to run a single test case ([using the `-k` option](https://docs.pytest.org/en/7.1.x/example/markers.html#using-k-expr-to-select-tests-based-on-their-name)).